website/static/js/pages/base-page.js

```diff
@@ -92,6 +93,101 @@ $(document).on('click', '.panel-heading', function(){
     }
 });
 
+//choose which emails/users to add to an account and which to deny
+function confirmEmails(emailsToAdd) {
+    if (emailsToAdd.length > 0) {
+        var confirmedEmailURL = window.contextVars.confirmedEmailURL;
+        var email = emailsToAdd.splice(0, 1)[0];
```

Comment:
It's generally a bad idea for functions to mutate the arguments passed to them. Why not use `Array.prototype.slice` (which doesn't mutate arrays) when you make the recursive calls instead?